MangaList/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,59 @@
+import model.Manga
+
+internal val library: MutableList<Manga> = mutableListOf()
```

Comment:
Nên cho tất tật cái file này thành một Controller, và đấy a đã nhắc rồi, cái mảng này của em anh có thể can thiệp vào ở bất cứ file nào. Không an toàn